sklearn/ensemble/tests/test_forest.py

```diff
@@ -10,14 +10,10 @@
 
 import pickle
 from collections import defaultdict
-from itertools import combinations
-from itertools import product
+from itertools import combinations, product
 
 import numpy as np
-from scipy.misc import comb
-from scipy.sparse import csr_matrix
-from scipy.sparse import csc_matrix
-from scipy.sparse import coo_matrix
```

Comment:
we typically don't import everything in one line to avoid too many changed lines when adding or removing some imports.

please revert